Vic2ToHoI4/Source/OutHoi4/OutTechnologies.cpp

```diff
@@ -5,32 +5,42 @@
 
 void HoI4::outputTechnology(const technologies& theTechnologies, std::ostream& output)
 {
+	const auto& technologiesByLimit = theTechnologies.getTechnologies();
+
 	output << "# Starting tech\n";
-	output << "set_technology = {\n";
-	for (const auto& technology: theTechnologies.getMainTechnologies())
-	{
-		output << "\t" << technology << " = 1\n";
-	}
-	output << "}\n";
-	output << "\n";
-	output << "if = {\n";
-	output << "\tlimit = { not = { has_dlc = \"Man the Guns\" } }\n";
-	output << "\tset_technology = {\n";
-	for (const auto& technology: theTechnologies.getNonMtgNavalTechnologies())
+
+	if (const auto& noLimitTechs = technologiesByLimit.find(""); noLimitTechs != technologiesByLimit.end())
```

Comment:
What exactly are you finding here?

Reply:
The technologies that have no limit (most of them). They have a different output format.